### ISSUE TITLE
Fargate tests will not run when custom JS is in a separate directory

### ISSUE THREAD

**Issue by ethan-coles** [artilleryio/artillery#2414]
Version info: 
```
Artillery: 2.0.3
Node.js:   v18.18.0
OS:        win32
```

Running this command: 
```
.\node_modules\.bin\artillery run-fargate --region us-east-1 --count 1 --cluster <cluster> --subnet-ids <subnet_id> -t <target_url> .\Test_001.yml
```

I expected to see this happen:

Have tests using fargate ECS clusters execute the same when running locally.

Instead, this happened:

Tests are not executed displaying `No measurements recorded during this period` on the console output. Checking the cloudwatch log on the tasks, I get this error thrown of not being able to find the custom JS processor file.
```
worker error, id: 1 Error: Cannot find module '/artillery/test_data/Functions/BaseProcessor.js'
Require stack:
- /artillery/packages/core/lib/runner.js
- /artillery/packages/core/index.js
- /artillery/packages/artillery/lib/artillery-global.js
- /artillery/packages/artillery/lib/platform/local/worker.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:119:18)
    at loadProcessor (/artillery/packages/core/lib/runner.js:87:23)
    at prepare (/artillery/packages/artillery/lib/platform/local/worker.js:109:18)
    at async MessagePort.onMessage (/artillery/packages/artillery/lib/platform/local/worker.js:50:5) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/artillery/packages/core/lib/runner.js',
    '/artillery/packages/core/index.js',
    '/artillery/packages/artillery/lib/artillery-global.js',
    '/artillery/packages/artillery/lib/platform/local/worker.js'
  ]
```

This works as expected when having all of my JS files in the same directory as the tests or if I run the above command on a linux machine. It leads me to believe that there may be an issue with the directory symbols between the 2 operating systems during the test bundling process when running the command on Windows for ECS linux instances. Issue is similar to #2092.

```
Windows Powershell

Test bundle contents:
┌─────────────────────────────────────────────────────────────┐
│ Name                                      │ Type    │ Notes │
├───────────────────────────────────────────┼─────────┼───────┤
│ Test_001.yml                              │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions\BaseProcessor.js                │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions\Action_User.js                  │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions\Helpers.js                      │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ appsettings.json                          │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ package.json                              │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ package-lock.json                         │ file    │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery                                 │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery-plugin-metrics-by-endpoint      │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery-pro                             │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ redis                                     │ package │       │
└───────────────────────────────────────────┴─────────┴───────┘

Linux Shell

Test bundle contents:
┌───────────────────────────────────────────┬─────────┬───────┐
│ Name                                      │ Type    │ Notes │
├───────────────────────────────────────────┼─────────┼───────┤
│ Test_001.yml                              │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions/BaseProcessor.js                │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions/Action_User.js                  │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ Functions/Helpers.js                      │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ appsettings.json                          │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ package.json                              │ file    │       │
├───────────────────────────────────────────┼─────────┤       │
│ package-lock.json                         │ file    │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery                                 │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery-plugin-metrics-by-endpoint      │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ artillery-pro                             │ package │       │
├───────────────────────────────────────────┼─────────┼───────┤
│ redis                                     │ package │       │
└───────────────────────────────────────────┴─────────┴───────┘
```

**Comment by ethan-coles** [comment]
Yes, that's correct.

**Comment by bernardobridge** [comment]
A fix for this will be released on version 2.0.5, coming this Monday.
